content/step-definitions.md

```diff
@@ -203,3 +203,33 @@ Given /Three (.*) mice/ do |disability|
 end
 ```
 {{% /text %}}
+
+### Nested Steps
+
+In an effort to keep your code DRY, you may be tempted to utilize Cucumber's feature of calling step definitions from other steps: **DON'T**. While this may seem like a useful feature to utilize, it creates bad code smells and can make things like readability difficult and stack traces terrible. Instead, in places where you're calling steps from other steps, use those as opportunities to refactor your code to create helper methods to use instead.
```

Comment:
Maybe explain "DRY" (for non-developers)?

Reply:
AFAIK nested steps are only possible in Ruby, so mark this text block with {{% text "ruby" %}} {{% /text %}}
Also, java & js examples will not be needed ;)

Reply:
Something to consider: As this is (also) a Best practice / Anti-pattern, maybe it should be in best-practices rather than step-definitions? On the other hand, this would probably the place where people read about how to write step definitions, so this might actually be the best place.

Reply:
Also here, see if you can make the sentences shorter & clearer by splitting them or removing words (i.e. duplicate "instead" in last sentence).

Reply:
* I'll rephrase and do a link to the wiki article on DRY. :)
* While not a part of the official Cucumber repo, the implementation of [Cucumber in Go also allows for nested steps.](https://github.com/DATA-DOG/godog/issues/110) Perhaps when trying to inform folks about this, I can just point them to the Ruby version of it instead.
* I'm open to adding a note to the best-practice page too. I'll do a different PR for that once we get this settled.
* Thanks for catching the duplicate instead! Ugh. I'll work on rephrasing, thanks!

Reply:
Great! Currently the polyglot pages have tabs for Java/Javascript/Ruby. Not sure what to do about the other language implementations, as they don't have a place in the nw menu structure anymore...?